src/helper/extractMetadata.ts

```diff
@@ -109,3 +120,7 @@ const formatTime = (time: string, language: string, timeFormat: DateTime): strin
   }
   return undefined
 }
+
+export const getDicomTagDescription = (otherInformation: object) => {
```

Comment:
also here it might be a bit easier to understand the code if you choose a name that is more descriptive

if I understand your code properly, the object `otherInformation` can be any list/dictionary and the function returns the descriptions of all items in that list or dictionary?